Stp.TestingApi/Controllers/TestCategoryController.cs

```diff
@@ -46,15 +80,35 @@ public IActionResult UpdateCategoryName(long categoryId, [FromBody]string name)
         [HttpPut(nameof(MoveCategory))]
         public IActionResult MoveCategory(MoveCategoryCommand cmd)
         {
-            throw new NotImplementedException();
+            var category = _db.TestCategoryList.Find(cmd.CategoryId);
+
+            if (category == null)
+            {
+                return BadRequest($"Test category with id={cmd.CategoryId} not found");
+            }
+
+            category.Position = cmd.Position;
+            category.ParentId = cmd.ParentCategoryId;
```

Comment:
It is also necessary to change the Position of other items within parent node (moving one item affecting positions of all sibling items).